### ISSUE TITLE
Error after checkout

### ISSUE THREAD

**Issue by nihar53** [Attendize/Attendize#198]
After i check out i get this error "Whoops!, it looks like the server returned an error. Please try again, or contact the webmaster if the problem persists."

**Comment by jmowatt** [comment]
Hi @nihar53 

This can be resolved by setting

MAIL_FROM_ADDRESS
MAIL_FROM_NAME

in your .env

Hope this helps

James

**Comment by nihar53** [comment]
That is already set in my .env, when i click the checkout button it does not show the ticket or go to the payment process page, it just shows the error above.

**Comment by jmowatt** [comment]
Have you got error logs?

**Comment by nihar53** [comment]
public_html/ticket/public/error_log:
#37 public_html/ticket/bootstrap/cache/compiled.php(9980): call_user_func_array(Array, Array)
#38 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 public_html/ticket/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#40 public_html/ticket/bootstrap/cache/compiled.php(11969): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 public_html/ticket/bootstrap/cache/compiled.php(9980): call_user_func_array(Array, Array)
#43 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 public_html/ticket/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#45 public_html/ticket/bootstrap/cache/compiled.php(13165): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#46 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 public_html/ticket/bootstrap/cache/compiled.php(9980): call_user_func_array(Array, Array)
#48 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#49 public_html/ticket/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#50 public_html/ticket/bootstrap/cache/compiled.php(13102): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#51 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#52 public_html/ticket/bootstrap/cache/compiled.php(9980): call_user_func_array(Array, Array)
#53 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#54 public_html/ticket/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#55 public_html/ticket/bootstrap/cache/compiled.php(3283): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#56 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#57 public_html/ticket/bootstrap/cache/compiled.php(9980): call_user_func_array(Array, Array)
#58 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 public_html/ticket/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#60 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#61 public_html/ticket/bootstrap/cache/compiled.php(9965): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#62 public_html/ticket/bootstrap/cache/compiled.php(2363): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#63 public_html/ticket/bootstrap/cache/compiled.php(2347): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#64 public_html/ticket/public/index.php(52): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#65 {main}

**Comment by nihar53** [comment]
That is the error log

**Comment by jacobcat** [comment]
The error log you posted started with #37... can you paste all the way to #0 including the error message it generated?

**Comment by Tyga76** [comment]
Hi, experiencing the same bug. Only free events go through succesfully.

**Comment by Turupawn** [comment]
Hi, I have this error too.
- I'm using the version downloaded [here](https://www.attendize.com/documentation.php#download) today
- In the storage/logs directory there's only the .gitignore file and no logs. Does anyone knows why?
- I was able to configure the email, I was able to send some email user invites

Help please :sob:

**Comment by Tyga76** [comment]
@Turupawn - is your event free? if not you have to have at least one payment gateway. Thats what resolved it for me.

**Comment by Turupawn** [comment]
Thanks for your help @Tyga76!
I set up a payment gateway. To do so, I had to fix the getGateway function as described on #155.
![](https://docs.google.com/uc?id=0B1epcuwQhxeUeW1wSmZ5NEVTVGM)

But it still doesn't works :cry: 

Some additional info that may help:

- I get the error installing both the zip and manual installation
- I'm using mysql, are you guys using that database too?
- I'm using paypal, are you using it too?

**Comment by daveearley** [comment]
Hi @Turupawn 

In your **.env** and you set **APP_DEBUG=true**. Then you will be able to see more detailed errors.

Most checkout issues are related to WKHTML2PDF driver files not being executable or being incorrectly set. You can find more info on that here: [https://attendize.com/documentation.php#troubleshooting](https://attendize.com/documentation.php#troubleshooting)

Also, make sure any settings in the **.env** file which contain a space are wrapped in double quotes.

Cheers

**Comment by njabulo-ivan** [comment]
Hi. To fix it in my server I had to do the following:

1. php artisan vendor:publish
2. apt-get install libxrender1 libxext6 

Found the solution in Troubleshooting section of this project https://github.com/NitMedia/wkhtml2pdf

**Comment by modifiedcontent** [comment]
I get this 'Whoops!' error with the Widget HTML Embed Code version of the ticket registration form, but if I use the normal (ugly) event url page generated by the application, registration works fine; payment goes through via the Stripe gateway, confirmation emails get send, etc. 

So in my case there are no problems with either email setup, pdf generation or the payment gateway. The embed widget seems to cause the problem. 

I don't get any error logs with App_DEBUG=true; nothing shows up in /storage/logs or error_log. So I'm not sure how to troubleshoot this embed problem. Is it some path mismatch? Javascript conflicts? Something else? When I embed the iframe widget on an empty php page, without head or any other potentially conflicting scripts, I still get the same 'Whoops!' error, so I guess there is a path/address problem?

The embed widget worked fine on the same websites/server two years ago, before "The Event" put my events business on hold. What changed? Anything else I should update? I already updated to Attendize-2.6.0.
